### ISSUE TITLE
allow excluding modules from result list

### ISSUE THREAD

**Issue by Znerole** [codepunkt/webpack-license-plugin#213]
I'm working on a project where we are using multiple in-house modules which do not have a license (less so an SPDX license), which currently prevents me from using this otherwise great plugin. It would be nice to be able to exclude modules from the license check/aggregation.

I think this could be married with the "licenseOverrides" map by using a special "ignore" string as license, e.g.:

```
"licenseOverrides": {
   "@some-company/cool-inhouse-module@1.2.3": "ignore",
}
```

**Comment by codepunkt** [comment]
Sounds great, will add an option for this. 👍 
Not sure about the details though - combining it with `licenseOverrides` seems smart, but i'd rather be verbose and have an own option for that.

**Comment by codepunkt** [comment]
@Znerole published as `4.1.0` with new option `excludedPackageTest`. Feedback is welcome!
